pymc3/distributions/continuous.py

```diff
@@ -1452,7 +1452,7 @@ class InverseGamma(PositiveContinuous):
     """
 
     def __init__(self, alpha, beta=1, *args, **kwargs):
-        super(InverseGamma, self).__init__(*args, **kwargs)
+        super(InverseGamma, self).__init__(*args, defaults=['mode'], **kwargs)
```

Comment:
You should probably use a tuple rather than a list because crazy things can happen with lists as default arguments.